README.md

```diff
@@ -2,16 +2,21 @@
 
 ## TODO
 
-* change the port to 8080!
-* setup docker
 * implement requirements
+* maybe add scripts to test the min load requirements
+* maybe add dev docker setup (via devcontainers or just MIX_ENV=dev)
 * profit!
 
 ## Setup
 
 ### Docker
 
-TBA
+TODO: check if `docker-compose run web "/app/bin/migrate"` is required (after migrations are added)
+
+* Run `docker-compose up -d`
+* Access the app at http://localhost:8080
+
+Note: this boots the app in prod environment.
```

Comment:
I chose prod setup in hope that I will later write script(s) to test this requirement:

> Your submission should be able to handle 5 requests per second to the
submission form, and 25 requests per second to the redirect endpoint, so choose a tech stack and design with this in mind. We will be testing these on current developer laptops (apple M1 Max or core i5+, 16 GB+ RAM).

This is the closest to prod we can get on a local machine (in real prod, the DB would be hosted on separate machine and the only thing needed to push the app would be Dockerfile.

I might still provide more dev-friendly setup, where all the Elixir tooling is available in the app container.